### ISSUE TITLE
Support sorting by date in studies/series tables

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#604]
As pointed out by a user in this thread https://discourse.canceridc.dev/t/tcga-gbm-tutorial-notebook/162, some cases contain multiple studies (and TCGA-GBM collection contains cases with over 10 studies!), and sorting by date is a much needed feature.

**Comment by G-White-ISB** [comment]
I thought dates are randomized/meaningless as part of the de-identification process

**Comment by fedorov** [comment]
No, they are most definitely not meaningless. It depends on the program, I guess, but for longitudinally collected data the dates are shifted to maintain temporal context.

**Comment by fedorov** [comment]
@G-White-ISB I see data in dev, but not in test. What is the timeline for the release of this feature?

**Comment by s-paquette** [comment]
@fedorov @pgundluru This is available in test or production for testing purposes. Not sure how it slipped through the cracks but I've added it to the milestone for testing and closure.

**Comment by fedorov** [comment]
I did clear cache prior to testing

**Comment by s-paquette** [comment]
Got it--will move to Sprint 27.

**Comment by G-White-ISB** [comment]
Testing needed in dev. Sorting by date in Study table should work. Also, when sorting by CASE ID, the table is sorted secondarily by study date (ascending).
